# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/svg/crashtests/1600855.html
# Commit: a9c150ebbf2b
# Full Hash: a9c150ebbf2b73b277790b5dd04a8704127fb215
# Author: Cameron McCormack <cam@mcc.id.au>
# Date: 2019-12-04 09:56:40
# Regressor Bug: 1600855
# File Overlap Count: 2
# Description:
#   Bug 1600855 - Avoid SVG text calling gfxTextRun::GetAdvanceWidth for ranges that cross text frame boundaries. r=longsonr
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D55600
# ==============================================================================

diff -r bc0035aa57fe -r a9c150ebbf2b layout/svg/crashtests/1600855.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/layout/svg/crashtests/1600855.html	Tue Dec 03 06:12:39 2019 +0000
@@ -0,0 +1,8 @@
+<script>
+window.onload = () => {
+  a.append(String.fromCodePoint(71341))
+}
+</script>
+<font style="word-spacing: 31pc">
+<svg>
+<text id="a">Text</text>